bedrock/firefox/templates/firefox/whatsnew/whatsnew-50.html

```diff
@@ -0,0 +1,44 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% set send_to_device_message_set = 'fx-50-whatsnew' %}
+{% set app_store_url = 'https://app.adjust.com/2uo1qc?campaign=whats_new&adgroup=iOS_App_Store&creative=badge_en&fallback=https%3A%2F%2Fitunes.apple.com%2Fapp%2Fapple-store%2Fid989804926%3Fpt%3D373246%26ct%3Dadjust_tracker%26mt%3D8' %}
+{% set play_store_url = 'https://app.adjust.com/2uo1qc?campaign=whats_new&adgroup=Google_Play_Store&creative=badge_en' %}
+
+{% extends "firefox/mobile-download-desktop.html" %}
```

Comment:
I wasn't sure if they needed to be present before the `extends`. I see now that they do not. 😁